src/drt/src/TritonRoute.cpp

```diff
@@ -942,6 +942,19 @@ int TritonRoute::main()
     }
   }
   initDesign();
+  bool has_routable_nets = false;
+  for (auto net : db_->getChip()->getBlock()->getNets()) {
+    if (net->getITerms().size() + net->getBTerms().size() > 1) {
+      has_routable_nets = true;
+      break;
+    }
+  }
+  if (!has_routable_nets) {
+    logger_->error(DRT,
```

Comment:
I think it might be better to give a warning and return.  It isn't strictly illegal.  Same in grt

Reply:
ok I'll do the change. Thanks.